Fails to publish to the Wiki

It seems the latest PR that was merged did not work as expected, it failed to publish to the Wiki when dogfooding itself. It will hopefully be resolved by PR #80.

See error https://dev.azure.com/dsccommunity/DscResource.DocGenerator/_build/results?buildId=4801&view=logs&j=5d0a9c4e-8741-5118-af45-406a30fe661c&t=bda0b158-a237-516f-c1d0-31339587c5b5&l=146

Raw output here (if the link above do not work):
```
2021-06-19T15:25:33.0578840Z Publishing Wiki content.
2021-06-19T15:25:33.2106611Z ERROR: Exception calling "Start" with "0" argument(s): "No such file or directory"
2021-06-19T15:25:33.2107271Z At /home/vsts/work/1/s/output/DscResource.DocGenerator/0.9.0/DscResource.DocGenerator.psm1:1402 char:13
2021-06-19T15:25:33.2108430Z +         if ($process.Start() -eq $true)
2021-06-19T15:25:33.2108871Z +             ~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-19T15:25:33.2151472Z At /home/vsts/work/1/s/source/tasks/Publish_GitHub_Wiki_Content.build.ps1:102 char:1
```

_Originally posted by @johlju in https://github.com/dsccommunity/DscResource.DocGenerator/issues/80#issuecomment-864421851_

https://dev.azure.com/dsccommunity/DscResource.DocGenerator/_build/results?buildId=4854&view=logs&j=5d0a9c4e-8741-5118-af45-406a30fe661c&t=bda0b158-a237-516f-c1d0-31339587c5b5&l=143

```
2021-06-26T13:45:53.5310573Z ===============================================================================
2021-06-26T13:45:53.5313104Z 			PUBLISH GITHUB WIKI CONTENT
2021-06-26T13:45:53.5354665Z This task publishes documentation to a GitHub Wiki repository.
2021-06-26T13:45:53.5356775Z -------------------------------------------------------------------------------
2021-06-26T13:45:53.5360691Z   /publish/Publish_GitHub_Wiki_Content
2021-06-26T13:45:53.5363474Z   /home/vsts/work/1/s/source/tasks/Publish_GitHub_Wiki_Content.build.ps1:102
2021-06-26T13:45:53.5364689Z 
2021-06-26T13:45:53.5418939Z 	OutputDirectory       = '/home/vsts/work/1/s/output'
2021-06-26T13:45:53.5481116Z 	Built Module Manifest         = '/home/vsts/work/1/s/output/DscResource.DocGenerator/0.9.0/DscResource.DocGenerator.psd1'
2021-06-26T13:45:53.5515675Z 	Built Module Base             = '/home/vsts/work/1/s/output/DscResource.DocGenerator/0.9.0'
2021-06-26T13:45:53.5567424Z 	Module Version                = '0.9.0-preview0006'
2021-06-26T13:45:53.5568160Z 	Module Version Folder         = '0.9.0'
2021-06-26T13:45:53.5569129Z 	Pre-release Tag               = 'preview0006'
2021-06-26T13:45:53.5569487Z 	Project Path                  = /home/vsts/work/1/s
2021-06-26T13:45:53.5569805Z 	Project Name                  = DscResource.DocGenerator
2021-06-26T13:45:53.5571644Z 	Source Path                   = /home/vsts/work/1/s/source
2021-06-26T13:45:53.5572612Z 	Built Module Base             = /home/vsts/work/1/s/output/DscResource.DocGenerator/0.9.0
2021-06-26T13:45:53.5599817Z Set GitHubConfigUserName to dscbot
2021-06-26T13:45:53.5621623Z Set GitHubConfigUserEmail to dsccommunity@outlook.com
2021-06-26T13:45:53.7074666Z ERROR: Exception calling "Start" with "0" argument(s): "No such file or directory"
2021-06-26T13:45:53.7075953Z At /home/vsts/work/1/s/output/DscResource.DocGenerator/0.9.0/DscResource.DocGenerator.psm1:2393 char:13
2021-06-26T13:45:53.7077240Z +         if ($process.Start() -eq $true)
2021-06-26T13:45:53.7078126Z +             ~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-26T13:45:53.7112605Z At /home/vsts/work/1/s/source/tasks/Publish_GitHub_Wiki_Content.build.ps1:102 char:1
2021-06-26T13:45:53.7113339Z + task Publish_GitHub_Wiki_Content {
2021-06-26T13:45:53.7113882Z + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-26T13:45:53.7144142Z At /home/vsts/work/1/s/build.ps1:211 char:13
2021-06-26T13:45:53.7145201Z +             task $Workflow $WorkflowItem
2021-06-26T13:45:53.7145744Z +             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-26T13:45:53.7258901Z Build FAILED. 4 tasks, 1 errors, 0 warnings 00:00:28.4099807
2021-06-26T13:45:53.7864835Z [91mException: [0m/home/vsts/work/1/s/output/DscResource.DocGenerator/0.9.0/DscResource.DocGenerator.psm1:2393
2021-06-26T13:45:53.7866832Z [96mLine |
2021-06-26T13:45:53.7868354Z [96m2393 | [0m         if ([96m$process.Start() -eq $true[0m)
2021-06-26T13:45:53.7869958Z [96m     | [91m             ~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-26T13:45:53.7871166Z [91m[96m     | [91mException calling "Start" with "0" argument(s): "No such file
2021-06-26T13:45:53.7872119Z [96m     | [91mor directory"
2021-06-26T13:45:53.7872661Z [0m
2021-06-26T13:45:53.8042780Z ##[error]PowerShell exited with code '1'.
```

I did a debug branch (https://github.com/dsccommunity/DscResource.DocGenerator/compare/main...johlju:debug-publish-wiki) to try to reproduce it to help figure this out. 

Below is the output for the debug run.

```plaintext
2021-06-26T15:02:55.2831954Z ##[section]Starting: Run Publish_GitHub_Wiki_Content
2021-06-26T15:02:55.3058866Z ==============================================================================
2021-06-26T15:02:55.3059170Z Task         : PowerShell
2021-06-26T15:02:55.3059450Z Description  : Run a PowerShell script on Linux, macOS, or Windows
2021-06-26T15:02:55.3059735Z Version      : 2.186.0
2021-06-26T15:02:55.3059964Z Author       : Microsoft Corporation
2021-06-26T15:02:55.3060271Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/powershell
2021-06-26T15:02:55.3060623Z ==============================================================================
2021-06-26T15:02:58.3857055Z Generating script.
2021-06-26T15:02:58.4708129Z ========================== Starting Command Output ===========================
2021-06-26T15:02:58.5127605Z ##[command]"C:\Program Files\PowerShell\7\pwsh.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'D:\a\_temp\5a6a23e0-1607-42c2-8d32-15d63a85381e.ps1'"
2021-06-26T15:02:59.4344532Z [pre-build] Starting Build Init
2021-06-26T15:02:59.4869940Z [pre-build] Prepending 'D:\a\1\s\output\RequiredModules' folder to PSModulePath
2021-06-26T15:02:59.8105641Z [pre-build] Prepending 'D:\a\1\s\output' folder to PSModulePath
2021-06-26T15:02:59.8190864Z [build] Starting build with InvokeBuild.
2021-06-26T15:03:00.2254069Z [build] Parsing defined tasks
2021-06-26T15:03:00.2316395Z [build] Loading Configuration from D:\a\1\s\build.yaml
2021-06-26T15:03:02.8814066Z Importing tasks from module Sampler.GitHubTasks
2021-06-26T15:03:04.2516714Z 	 Loading New-Release.GitHub.build.Sampler.GitHubTasks.ib.tasks...
2021-06-26T15:03:06.3996502Z Importing tasks from module Sampler
2021-06-26T15:03:06.4239043Z 	 Loading Build-Module.ModuleBuilder.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4240371Z 	 Loading Clean.ModuleBuilder.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4241275Z 	 Loading DeployAll.PSDeploy.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4242419Z 	 Loading DscResource.Test.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4243434Z 	 Loading generateHelp.PlatyPS.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4244196Z 	 Loading GuestConfig.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4244916Z 	 Loading Invoke-Pester.pester.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4246248Z 	 Loading JaCoCo.coverage.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4246927Z 	 Loading release.module.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4247637Z 	 Loading Set-BuildEnvironment.BuildHelpers.build.Sampler.ib.tasks...
2021-06-26T15:03:06.4248299Z 	 Loading Set-SamplerTaskVariable...
2021-06-26T15:03:07.8687715Z Adding Workflow from configuration:
2021-06-26T15:03:07.9577723Z   +-> publish
2021-06-26T15:03:07.9695737Z   +-> pack
2021-06-26T15:03:07.9762363Z   +-> build
2021-06-26T15:03:07.9764016Z   +-> test
2021-06-26T15:03:07.9764904Z   +-> merge
2021-06-26T15:03:07.9767591Z   +-> .
2021-06-26T15:03:07.9836173Z   +-> hqrmtest
2021-06-26T15:03:07.9843486Z [build] Executing requested workflow: Publish_GitHub_Wiki_Content
2021-06-26T15:03:08.0385219Z Build Publish_GitHub_Wiki_Content D:\a\1\s\build.ps1
2021-06-26T15:03:08.0393899Z Redefined task '.'.
2021-06-26T15:03:08.0670953Z 
2021-06-26T15:03:08.0676810Z ===============================================================================
2021-06-26T15:03:08.0821867Z 			PUBLISH GITHUB WIKI CONTENT
2021-06-26T15:03:08.1020313Z This task publishes documentation to a GitHub Wiki repository.
2021-06-26T15:03:08.1022320Z -------------------------------------------------------------------------------
2021-06-26T15:03:08.1023857Z   /Publish_GitHub_Wiki_Content
2021-06-26T15:03:08.1134104Z   D:\a\1\s\source\tasks\Publish_GitHub_Wiki_Content.build.ps1:102
2021-06-26T15:03:08.1134398Z 
2021-06-26T15:03:08.1311750Z 	OutputDirectory       = 'D:\a\1\s\output'
2021-06-26T15:03:08.1606952Z 	Built Module Manifest         = 'D:\a\1\s\output\DscResource.DocGenerator\0.9.0\DscResource.DocGenerator.psd1'
2021-06-26T15:03:08.1631273Z 	Built Module Base             = 'D:\a\1\s\output\DscResource.DocGenerator\0.9.0'
2021-06-26T15:03:08.1815087Z 	Module Version                = '0.9.0-debug'
2021-06-26T15:03:08.1833571Z 	Module Version Folder         = '0.9.0'
2021-06-26T15:03:08.1845630Z 	Pre-release Tag               = 'debug'
2021-06-26T15:03:08.1868880Z 	Project Path                  = D:\a\1\s
2021-06-26T15:03:08.1874445Z 	Project Name                  = DscResource.DocGenerator
2021-06-26T15:03:08.1886378Z 	Source Path                   = D:\a\1\s\source
2021-06-26T15:03:08.1935533Z 	Built Module Base             = D:\a\1\s\output\DscResource.DocGenerator\0.9.0
2021-06-26T15:03:08.1936301Z Set GitHubConfigUserName to dscbot
2021-06-26T15:03:08.1936827Z Set GitHubConfigUserEmail to dsccommunity@outlook.com
2021-06-26T15:03:09.1856708Z DEBUG: Invoking Git 'remote get-url origin'.
2021-06-26T15:03:09.2654584Z 	Wiki Output Path              = D:\a\1\s\output\WikiContent
2021-06-26T15:03:09.2657404Z Publishing Wiki content.
2021-06-26T15:03:09.2728800Z VERBOSE: Creating a temporary working directory.
2021-06-26T15:03:09.2872287Z VERBOSE: Cloning the Wiki Git Repository 'https://github.com/dsccommunity/DscResource.DocGenerator.wiki.git'.
2021-06-26T15:03:09.2880544Z DEBUG: Invoking Git 'clone https://github.com/dsccommunity/DscResource.DocGenerator.wiki.git C:\Users\VssAdministrator\AppData\Local\Temp\eocrxt2p.1v5 --quiet'.
2021-06-26T15:03:09.4155084Z VERBOSE: Failed to clone wiki. Ensure the feature is enabled and the first page has been created.
2021-06-26T15:03:09.4170201Z DEBUG: Wiki clone URL 'https://github.com/dsccommunity/DscResource.DocGenerator.wiki.git'
2021-06-26T15:03:09.4188091Z DEBUG: 
2021-06-26T15:03:09.4206652Z DEBUG: 
2021-06-26T15:03:09.4219033Z DEBUG: git exit code: '-1'
2021-06-26T15:03:09.6473614Z ERROR: The process cannot access the file 'C:\Users\VssAdministrator\AppData\Local\Temp\eocrxt2p.1v5' because it is being used by another process.
2021-06-26T15:03:09.6513057Z At D:\a\1\s\output\DscResource.DocGenerator\0.9.0\DscResource.DocGenerator.psm1:3080 char:9
2021-06-26T15:03:09.6529657Z +         Remove-Item -Path $tempPath -Recurse -Force
2021-06-26T15:03:09.6530076Z +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-26T15:03:09.6558876Z At D:\a\1\s\source\tasks\Publish_GitHub_Wiki_Content.build.ps1:102 char:1
2021-06-26T15:03:09.6686548Z + task Publish_GitHub_Wiki_Content {
2021-06-26T15:03:09.6687199Z + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-26T15:03:09.6712262Z Build FAILED. 1 tasks, 1 errors, 0 warnings 00:00:09.7322536
2021-06-26T15:03:09.7040205Z Remove-Item : The process cannot access the file 'C:\Users\VssAdministrator\AppData\Local\Temp\eocrxt2p.1v5' because it is being used by another process.
2021-06-26T15:03:09.7058430Z At D:\a\1\s\output\DscResource.DocGenerator\0.9.0\DscResource.DocGenerator.psm1:3080 char:9
2021-06-26T15:03:09.7059716Z +         Remove-Item -Path $tempPath -Recurse -Force
2021-06-26T15:03:09.7060199Z +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-06-26T15:03:09.7065908Z + CategoryInfo          : WriteError: (C:\Users\VssAdminis…l\Temp\eocrxt2p.1v5:DirectoryInfo) [Remove-Item], IOException
2021-06-26T15:03:09.7066660Z + FullyQualifiedErrorId : RemoveFileSystemItemIOError,Microsoft.PowerShell.Commands.RemoveItemCommand
2021-06-26T15:03:09.8197796Z ##[error]PowerShell exited with code '1'.
2021-06-26T15:03:09.9003606Z ##[section]Finishing: Run Publish_GitHub_Wiki_Content
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -48,12 +48,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Converted to public function.
   - Updated to use `System.Diagnostics.Process` for improved error handling.
   - Returns object, allowing caller to process result.
+  - `git` commands no longer use `--quiet` to populate returned object.
 
 ### Fixed
 
 - `Publish_GitHub_Wiki_Content`
   - Output message if `$GitHubToken` not specified which skips this task.
     Fixes [Issue 75](https://github.com/dsccommunity/DscResource.DocGenerator/issues/75)
+- `Invoke-Git`
+  - Set `$TimeOut` to Milliseconds
+    Fixes [Issue 84](https://github.com/dsccommunity/DscResource.DocGenerator/issues/84)
 
 ## [0.8.3] - 2021-04-10
 

--- a/source/Public/Invoke-Git.ps1
+++ b/source/Public/Invoke-Git.ps1
@@ -9,7 +9,7 @@
         The path to the git working directory.
 
     .PARAMETER Timeout
-        Seconds to wait for process to exit.
+        Milliseconds to wait for process to exit.
 
     .PARAMETER Arguments
         The arguments to pass to the Git executable.
@@ -20,9 +20,9 @@
         Invokes the Git executable to clone the specified repository to the working directory.
 
     .EXAMPLE
-        Invoke-Git -WorkingDirectory 'C:\SomeDirectory' -Arguments @( 'status' ) -TimeOut 10
+        Invoke-Git -WorkingDirectory 'C:\SomeDirectory' -Arguments @( 'status' ) -TimeOut 10000
 
-        Invokes the Git executable to return the status while having a 10 second timeout.
+        Invokes the Git executable to return the status while having a 10000 millisecond timeout.
 #>
 
 function Invoke-Git
@@ -37,7 +37,7 @@ function Invoke-Git
 
         [Parameter(Mandatory = $false)]
         [System.Int32]
-        $TimeOut = 120,
+        $TimeOut = 120000,
 
         [Parameter(ValueFromRemainingArguments = $true)]
         [System.String[]]

--- a/source/Public/Publish-WikiContent.ps1
+++ b/source/Public/Publish-WikiContent.ps1
@@ -116,7 +116,7 @@ function Publish-WikiContent
         Write-Verbose -Message ($script:localizedData.CloneWikiGitRepoMessage -f $WikiRepoName)
 
         $gitCloneResult = Invoke-Git -WorkingDirectory $tempPath.FullName `
-                            -Arguments @( 'clone', $wikiRepoName, $tempPath, '--quiet' )
+                            -Arguments @( 'clone', $wikiRepoName, $tempPath )
 
         if ($gitCloneResult.ExitCode -eq 0)
         {
@@ -152,7 +152,7 @@ function Publish-WikiContent
             Write-Verbose -Message ($localizedData.CommitAndTagRepoChangesMessage -f $ModuleVersion)
 
             $gitCommitResult = Invoke-Git -WorkingDirectory $tempPath.FullName `
-                                    -Arguments @( 'commit', '--message', "`"$($localizedData.UpdateWikiCommitMessage -f $ModuleVersion)`"", '--quiet' )
+                                    -Arguments @( 'commit', '--message', "`"$($localizedData.UpdateWikiCommitMessage -f $ModuleVersion)`"" )
 
             if ($gitCommitResult.ExitCode -eq 0)
             {
@@ -162,10 +162,10 @@ function Publish-WikiContent
                 Write-Verbose -Message $localizedData.PushUpdatedRepoMessage
 
                 $null = Invoke-Git -WorkingDirectory $tempPath.FullName `
-                            -Arguments @( 'push', 'origin', '--quiet' )
+                            -Arguments @( 'push', 'origin' )
 
                 $null = Invoke-Git -WorkingDirectory $tempPath.FullName `
-                            -Arguments @( 'push', 'origin', $ModuleVersion, '--quiet' )
+                            -Arguments @( 'push', 'origin', $ModuleVersion )
 
                 Write-Verbose -Message $localizedData.PublishWikiContentCompleteMessage
             }

--- a/tests/unit/public/Publish-WikiContent.Tests.ps1
+++ b/tests/unit/public/Publish-WikiContent.Tests.ps1
@@ -232,7 +232,7 @@ InModuleScope $script:moduleName {
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
                     $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq "$($mockPublishWikiContentParameters.ModuleVersion)"
+                    $Arguments[2] -like $mockPublishWikiContentParameters.ModuleVersion
                 } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Remove-Item -Exactly -Times 1 -Scope It

--- a/tests/unit/public/Publish-WikiContent.Tests.ps1
+++ b/tests/unit/public/Publish-WikiContent.Tests.ps1
@@ -227,7 +227,13 @@ InModuleScope $script:moduleName {
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
                     $Arguments[1] -eq 'origin'
-                } -Exactly -Times 2 -Scope It
+                } -Exactly -Times 1 -Scope It
+
+                Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
+                    $Arguments[0] -eq 'push' -and
+                    $Arguments[1] -eq 'origin' -and
+                    $Arguments[2] -eq "$($mockPublishWikiContentParameters.ModuleVersion)"
+                } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Remove-Item -Exactly -Times 1 -Scope It
             }

--- a/tests/unit/public/Publish-WikiContent.Tests.ps1
+++ b/tests/unit/public/Publish-WikiContent.Tests.ps1
@@ -226,13 +226,14 @@ InModuleScope $script:moduleName {
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
-                    $Arguments[1] -eq 'origin'
+                    $Arguments[1] -eq 'origin' -and
+                    $Arguments[2] -eq $null
                 } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
                     $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -like $mockPublishWikiContentParameters.ModuleVersion
+                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion
                 } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Remove-Item -Exactly -Times 1 -Scope It

--- a/tests/unit/public/Publish-WikiContent.Tests.ps1
+++ b/tests/unit/public/Publish-WikiContent.Tests.ps1
@@ -109,8 +109,7 @@ InModuleScope $script:moduleName {
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'commit' -and
                     $Arguments[1] -eq '--message' -and
-                    $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $mockPublishWikiContentParameters.ModuleVersion)`"" -and
-                    $Arguments[3] -eq '--quiet'
+                    $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $mockPublishWikiContentParameters.ModuleVersion)`""
                 } -Exactly -Times 0 -Scope It
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
@@ -123,15 +122,13 @@ InModuleScope $script:moduleName {
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
-                    $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq '--quiet'
+                    $Arguments[1] -eq 'origin'
                 } -Exactly -Times 0 -Scope It
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
                     $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion -and
-                    $Arguments[3] -eq '--quiet'
+                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion
                 } -Exactly -Times 0 -Scope It
 
                 Assert-MockCalled -CommandName Remove-Item -Exactly -Times 1 -Scope It
@@ -216,8 +213,7 @@ InModuleScope $script:moduleName {
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'commit' -and
                     $Arguments[1] -eq '--message' -and
-                    $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $mockPublishWikiContentParameters.ModuleVersion)`"" -and
-                    $Arguments[3] -eq '--quiet'
+                    $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $mockPublishWikiContentParameters.ModuleVersion)`""
                 } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
@@ -230,15 +226,13 @@ InModuleScope $script:moduleName {
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
-                    $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq '--quiet'
+                    $Arguments[1] -eq 'origin'
                 } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
                     $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion -and
-                    $Arguments[3] -eq '--quiet'
+                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion
                 } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Remove-Item -Exactly -Times 1 -Scope It
@@ -270,8 +264,7 @@ InModuleScope $script:moduleName {
                 } -ParameterFilter {
                         $Arguments[0] -eq 'commit' -and
                         $Arguments[1] -eq '--message' -and
-                        $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $ModuleVersion)`"" -and
-                        $Arguments[3] -eq '--quiet'
+                        $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $ModuleVersion)`""
                     }
             }
 
@@ -337,8 +330,7 @@ InModuleScope $script:moduleName {
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'commit' -and
                     $Arguments[1] -eq '--message' -and
-                    $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $mockPublishWikiContentParameters.ModuleVersion)`"" -and
-                    $Arguments[3] -eq '--quiet'
+                    $Arguments[2] -eq "`"$($localizedData.UpdateWikiCommitMessage -f $mockPublishWikiContentParameters.ModuleVersion)`""
                 } -Exactly -Times 1 -Scope It
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
@@ -351,15 +343,13 @@ InModuleScope $script:moduleName {
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
-                    $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq '--quiet'
+                    $Arguments[1] -eq 'origin'
                 } -Exactly -Times 0 -Scope It
 
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
                     $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion -and
-                    $Arguments[3] -eq '--quiet'
+                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion
                 } -Exactly -Times 0 -Scope It
 
                 Assert-MockCalled -CommandName Remove-Item -Exactly -Times 1 -Scope It

--- a/tests/unit/public/Publish-WikiContent.Tests.ps1
+++ b/tests/unit/public/Publish-WikiContent.Tests.ps1
@@ -227,13 +227,7 @@ InModuleScope $script:moduleName {
                 Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
                     $Arguments[0] -eq 'push' -and
                     $Arguments[1] -eq 'origin'
-                } -Exactly -Times 1 -Scope It
-
-                Assert-MockCalled -CommandName Invoke-Git -ParameterFilter {
-                    $Arguments[0] -eq 'push' -and
-                    $Arguments[1] -eq 'origin' -and
-                    $Arguments[2] -eq $mockPublishWikiContentParameters.ModuleVersion
-                } -Exactly -Times 1 -Scope It
+                } -Exactly -Times 2 -Scope It
 
                 Assert-MockCalled -CommandName Remove-Item -Exactly -Times 1 -Scope It
             }

